app/templates/records/related_records.html

```diff
@@ -16,10 +17,10 @@
 {% block content %}
 <div class="tna-section">
   <div class="tna-container">
-    <div class="tna-column tna-column--width-2-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
-      <h1 class="tna-heading-xl">Related records for {{ pageTitle }}</h1>
+    <div class="tna-column tna-column--width-3-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
+
+      {{ related_records_page(record) }}
```

Comment:
Does this need to be a macro? I only ask because there isn't much opportunity to move it around on the page or reuse it.

Happy to run with this if you like.

Reply:
I think this is just going to be a standalone page for the moment, and the related records section on the detail page may be coming out. 

Let's go with this for the meantime - we'll have to revisit this when the mechanics of how we pull in related records are figured out..